src/rust/lib/graph-editor/src/component/operator.rs

```diff
@@ -0,0 +1,271 @@
+//! Provides generic operations that can be applied to UI components.
```

Comment:
I dont like `component::operator` name. I was not expecting this content when reading the file name. Im still not sure what the proper name is tohugh.